src/coverlet.msbuild.tasks/coverlet.msbuild.props

```diff
@@ -19,7 +19,7 @@
     <ExcludeAssembliesWithoutSources Condition="$(ExcludeAssembliesWithoutSources) == ''"></ExcludeAssembliesWithoutSources>
   </PropertyGroup>
   <PropertyGroup>
-    <CoverletToolsPath Condition="'$(CoverletToolsPath)' == '' and '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tasks\net6.0\</CoverletToolsPath>
-    <CoverletToolsPath Condition="'$(CoverletToolsPath)' == '' and '$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tasks\netstandard2.0\</CoverletToolsPath>
+    <CoverletToolsPath Condition="$(TargetFrameworkIdentifier) == '.NETCoreApp'">$(MSBuildThisFileDirectory)..\tasks\net6.0\</CoverletToolsPath>
```

Comment:
we're building multi-tfm https://github.com/coverlet-coverage/coverlet/blob/783c482bbb1d59e02e6859fd5397ebda95774f3c/src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj#L5

I think we load the net6.0 or the netstandard if the msbuild node is netfx @Bertk  can you confirm you did this change in past? I don't recall if we were compiling multi-tfm or not

Reply:
Yes the `coverlet.msbuild` package contains the assemblies for both. But it doesn't look like the `net6` assemblies are used at all. No matter if the using application is core or netfx. Thus we then could also move `coverlet.msbuild.tasks` back to only being netstandard.

Reply:
agree let's stay compat

Reply:
I am happy to test it in .net framework projects if there is a beta or something like that...